SocialCareCaseViewerApi/V1/Boundary/Requests/QueryCaseSubmissionsRequest.cs

```diff
@@ -1,46 +1,48 @@
 using System;
 using System.Collections.Generic;
-using System.Text.Json.Serialization;
 using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
 
 namespace SocialCareCaseViewerApi.V1.Boundary.Requests
 #nullable enable
 {
     public class QueryCaseSubmissionsRequest
     {
-        [JsonPropertyName("formId")]
```

Comment:
These were never JSON properties, whoops.
Somehow .net doesn't mind anyway but I'd rather the attribute actually be correct.